src/tests.c

```diff
@@ -28,7 +29,8 @@
 
 #define CONDITIONAL_TEST(cnt, nam) if (COUNT < (cnt)) { printf("Skipping %s (iteration count too low)\n", nam); } else
 
-static int COUNT = 64;
+static const int DEFAULT_COUNT = 64;
+static int COUNT = DEFAULT_COUNT;
```

Comment:
MSVC doesn't like that line apparently because `DEFAULT_COUNT` is not a constant, which may be true by a strict reading of the standard... 

Maybe try `#defining DEFAULT_COUNT` instead, or set `COUNT = DEFAULT_COUNT;` only at the beginning of `main()`.

(By the way, this shows that CI on MSVC will timeout after 60 min in case of failure instead of just aborting. This will be fixed by https://github.com/bitcoin-core/secp256k1/pull/1209/commits/e4330341bd648e93b60fe70c631e311a98bce549)